test/echo-tcp-client/main.cpp

```diff
@@ -41,6 +36,7 @@ void terminate(bool status, TCPEchoClient* client);
 
 char out_buffer[] = "Hello World\n";
 char buffer[256];
+char out_success[] = "{{success}}\n{{end}}\n";
```

Comment:
This string is confusing, is this just a payload used for test or actual control command?